setupext.py

```diff
@@ -8,17 +9,17 @@
 import tempfile
 from textwrap import dedent
 from concurrent.futures import ThreadPoolExecutor
-from distutils import log
 from distutils.ccompiler import CCompiler, new_compiler
-from distutils.errors import CompileError, LinkError
 from distutils.sysconfig import customize_compiler
 from subprocess import PIPE, Popen
 from sys import platform as _platform
 
 from pkg_resources import resource_filename
 from setuptools.command.build_ext import build_ext as _build_ext
 from setuptools.command.sdist import sdist as _sdist
+from setuptools.errors import CompileError, LinkError
 
+log = logging.getLogger("setupext")
```

Comment:
should we explicitly note that this is yt's setupext?

Reply:
I don't think there's any risk of collisions so I guess not ?